FreeRTOS-Plus/Source/FreeRTOS-Plus-TCP/History.txt

```diff
@@ -1,3 +1,47 @@
+Changes between V2.2.1 and V2.2.2
+	+ Add CBMC proofs for 4 functions.
+	+ Address ~1200 MISRA violations.
```

Comment:
This looks really alarming, but I expect most of the 1200 are the same small change repeated in lots of places.  Best to be a bit more generic and say something like "stricter MISRA compliance", or "updated to conform to additional MISRA rules [then list them].